packages/controller/test/lib/testAdapterHelpers.js

```diff
@@ -67,7 +67,7 @@ function register(it, expect, context) {
         this.timeout(3000);
 
         //Expecting a callback
-        context.adapter.checkPassword('claus', '1234').should.be.rejectedWith('checkPassword: no callback');
+        expect(context.adapter.checkPassword.bind('claus', '1234')).to.throw('checkPassword: no callback');
```

Comment:
This seems wrong. You're passing `'claus'` as the `this` argument. Just use an arrow function.

```suggestion
        expect(() => context.adapter.checkPassword('claus', '1234')).to.throw('checkPassword: no callback');
```

Reply:
There are some other instances of this in the file, just not changed this time.

Reply:
Totally true, good catch.